swap_meet/vendor.py

```diff
@@ -1,2 +1,86 @@
+from .item import Item
+
+
 class Vendor:
-    pass
\ No newline at end of file
+ 
+    def __init__(self, inventory=None):
+        self.inventory = [] if inventory is None else inventory
+ 
+    def add(self, item):
```

Comment:
Consider adding in a guard clause, aka a condition to check whether or not the `item` being appended is, in fact, an item.

This doesn't affect the tests nor was it required, but it never hurts to validate input when we can!